code/drasil-lang/Language/Drasil/Expr/Display.hs

```diff
@@ -0,0 +1,32 @@
+{-# LANGUAGE GADTs #-}
+
+module Language.Drasil.Expr.Display (DisplayExpr(..), Display(..),
+  defines, multiExpr, multiExprNE) where
+
+import qualified Data.List.NonEmpty as NE
+
+import Language.Drasil.Expr (Expr)
+
+data DisplayExpr where
+    AlgebraicExpr :: Expr -> DisplayExpr
+    Defines       :: DisplayExpr -> DisplayExpr -> DisplayExpr
+    MultiExpr     :: NE.NonEmpty DisplayExpr -> DisplayExpr
+
+
+class Display c where
+  toDispExpr :: c -> DisplayExpr
+
+instance Display Expr where
+  toDispExpr = AlgebraicExpr
+
+instance Display DisplayExpr where
+  toDispExpr = id
```

Comment:
I really wanted to place these in the `Classes` file, but I seem to run into circular dependency issues fairly quickly when I add in the 2 instances. I'll probably try to move it again though.

Reply:
Add a `DisplayClasses` file, i.e. `Language.Drasil.DisplayClasses`, and put the class there.

Reply:
Sounds good, just did this in 83dbb923653f43e7bf4212646c6e65d845b47bab